downstream/titles/aap-planning-guide/master.adoc

```diff
@@ -8,7 +8,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= {PlatformName} Planning Guide
+= Red Hat Ansible Automation Hub planning guide
```

Comment:
This is not a hub planning guide but rather the Red Hat Ansible Automation Platform planning guide